readme.md

```diff
@@ -31,5 +31,15 @@ Basically you call the script with a subcommand, e.g.:
 
     ./wine_docker.sh ellisys
     ./wine_docker.sh ellisys dump.btt
+    ./wine_docker.sh ebq_viewer dump.thd
     ./wine_docker.sh interactive   # default behavior. Gives you a shell inside
 
+# Troubleshooting
+
+* Ellysis crashes when you try to open / save file </br>
```

Comment:
This might be due to there not being a Desktop folder in the user's home directory in the prefix. The file dialog opens the Desktop folder by default, and if it's missing the program crashes.
Something along the lines of `/opt/wineprefix/drive_c/users/$USER/Desktop` must be created in order to avoid this.